.github/workflows/demo.yml

```
@@ -0,0 +1,281 @@
+  screengrabs:
+    needs: [prepare]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y aha util-linux make jq bat ttyd
+          sudo ln -s /usr/bin/batcat /usr/bin/bat
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)